spec/features/restaurants/view_restaurant_spec.rb

```diff
@@ -8,11 +8,21 @@
       sign_in_as user
     end
 
-    scenario 'I can see all restaurants' do
-      restaurant = FactoryGirl.create(:restaurant)
+    scenario 'I can see all restaurants on their appropriate pages i.e. paginate
+      works' do
+      next_page_restaurant = FactoryGirl.create(
+        :restaurant, name: "first restaurant"
+        )
+      restaurants = Array.new
+      25.times { |r| restaurants << FactoryGirl.create(:restaurant) }
```

Comment:
Unused block argument - `r`. You can omit the argument if you don't care about it.